<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marilia P. Gaiarsa, Sandra Rehan, Matthew Barbour, Quinn McFrederick" />


<title>Individual dietary specialization in a generalist bee varies across populations but has no effect on the richness of associated microbial communities</title>

<script src="ceratina_writeup_files/header-attrs-2.10/header-attrs.js"></script>
<script src="ceratina_writeup_files/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="ceratina_writeup_files/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="ceratina_writeup_files/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="ceratina_writeup_files/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="ceratina_writeup_files/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="ceratina_writeup_files/navigation-1.1/tabsets.js"></script>
<link href="ceratina_writeup_files/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="ceratina_writeup_files/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>




<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>



<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div id="header">



<h1 class="title toc-ignore">Individual dietary specialization in a generalist bee varies across populations but has no effect on the richness of associated microbial communities</h1>
<h4 class="author">Marilia P. Gaiarsa, Sandra Rehan, Matthew Barbour, Quinn McFrederick</h4>
<h4 class="date">March 2022, <em>The American Naturalist</em></h4>

</div>


<style>
body {
text-align: justify}
</style>
<div id="overview" class="section level1">
<h1>Overview</h1>
<p>We examine the occurrence of individual specialization in a solitary bee species, <em>Ceratina australensis</em> in three different wild populations from Australia. We use data previously published by <span class="citation">McFrederick and Rehan (2019)</span> on metabarcoding of the pollen contents of <em>Ceratina</em> nests to characterize the plant, bacteria, and fungi present in each individual nest. We show that bee individuals have marked dietary specialization and that the level of specialization varies across populations. In the most specialized population, we also show that individuals’ diet breadth was positively related to the richness of fungi, but not bacteria. Overall, individual specialization appeared to have a weak or negligible effect on the microbial richness of nests, suggesting that different mechanisms beyond environmental transmission may be at play regarding microbial acquisition in wild bees.</p>
<p>The analyses, raw data, and metadat are deposited in Zenodo and the github repo <a href="https://github.com/Magaiarsa/ceratinaIndDiet">ceratinaIndDiet</a>. Previously published sequence data are publicly available under NCBI/EMBL/DDBJ accession numbers SAMN08911168- SAMN08911424.</p>
<div id="data-metadata" class="section level2">
<h2>Data &amp; metadata</h2>
<p>The excel spreadsheet <code>rawData.xlsx</code> contains all raw data used in the analyses. In all of the tabs, each line represents a genus/ASV and the columns have information related to the metabarcoding (sequence ID, best taxonomic match, etc), as well as the number of reads found in each brood cell of each individual nest. There are four spreadsheets: <code>plant</code> (with the genus-level plant data), <code>plant_asv</code> (with the ASV plant data—see Online Supplement for details), <code>bact</code> (with the ASV bacteria data), and <code>fungi</code> (with the ASV fungi data). From the raw data, as described in the methods, we created matrices of nests (individuals) x taxonomic group for plants, plants ASV, bacteria, and fungi. We then used the plant matrix to calculate proportional similarity index for each individual <span class="citation">(Schoener 1968; Roughgarden 1979; Zaccarelli, Bolnick, and Mancinelli 2013)</span>, using the <code>PSicalc</code> function in the <code>RInSp</code> package <span class="citation">(Zaccarelli, Bolnick, and Mancinelli 2013)</span>. As a measure of resource overlap among individuals, we calculated <code>weighted.closeness</code> that individual in their plant network from the plant matrix, using the <code>weighted.closeness</code> function in the <code>bipartite</code> package <span class="citation">(Dormann, Gruber, and Fründ 2008)</span>.</p>
<p>In the file <code>metadata.csv</code> each line represents one individual/nest (<code>inds</code>) from each of the three different populations (<code>pop</code>). The column <code>degree</code> refers to the total number of plant genera present in the nest (richness), <code>weighted.closeness</code> is the closeness centrality, <code>psis</code> is the proportional similarity index. The next six columns are the rarefied richness of bacteria and fungi associated to the pollen, in three different rarefaction depths (see Methods and Online Supplement for details).</p>
<pre><code>## # A tibble: 38 × 11
##    inds  pop   degree weighted.closeness  psis bact.rich bact.rich.2
##    &lt;chr&gt; &lt;chr&gt;  &lt;dbl&gt;              &lt;dbl&gt; &lt;dbl&gt;     &lt;dbl&gt;       &lt;dbl&gt;
##  1 Q116  Q         14             0.0954 0.530      35.0        30.0
##  2 Q119  Q         12             0.113  0.647      NA          NA  
##  3 Q31   Q         13             0.0968 0.577      42.2        34.4
##  4 Q36   Q          8             0.121  0.817      15.8        14.3
##  5 Q55   Q          6             0.136  0.391      21.3        19.4
##  6 Q74   Q          5             0.121  0.640      40.5        33.3
##  7 Q77   Q          9             0.128  0.602      NA          NA  
##  8 Q89   Q          5             0.126  0.369      35.8        31.3
##  9 Q94   Q          7             0.131  0.742      48.5        37.6
## 10 S104  S          7             0.0593 0.493      21.7        19.1
## # … with 28 more rows, and 4 more variables: bact.rich.notrare &lt;dbl&gt;,
## #   fungi.rich &lt;dbl&gt;, fungi.rich.2 &lt;dbl&gt;, fungi.rich.notrare &lt;dbl&gt;</code></pre>
</div>
<div id="code" class="section level2">
<h2>Code</h2>
<p>The script <code>analyses.R</code> loads the metadata file above and details the different statistical analyses and recreates the plots from the main text.</p>
</div>
<div id="session-info" class="section level2">
<h2>Session Info</h2>
<pre class="r"><code>sessionInfo() </code></pre>
<pre><code>## R version 4.1.1 (2021-08-10)
## Platform: x86_64-pc-linux-gnu (64-bit)
## Running under: Ubuntu 20.04.3 LTS
## 
## Matrix products: default
## BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
## LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0
## 
## locale:
##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] forcats_0.5.1   stringr_1.4.0   purrr_0.3.4     readr_2.0.1    
##  [5] tidyr_1.1.3     tibble_3.1.3    tidyverse_1.3.1 ggplot2_3.3.5  
##  [9] dplyr_1.0.7     RInSp_1.2.4    
## 
## loaded via a namespace (and not attached):
##  [1] tidyselect_1.1.1  xfun_0.25         bslib_0.2.5.1     haven_2.4.3      
##  [5] colorspace_2.0-2  vctrs_0.3.8       generics_0.1.0    htmltools_0.5.1.1
##  [9] yaml_2.2.1        utf8_1.2.2        rlang_0.4.11      jquerylib_0.1.4  
## [13] pillar_1.6.2      glue_1.4.2        withr_2.4.2       DBI_1.1.1        
## [17] bit64_4.0.5       dbplyr_2.1.1      modelr_0.1.8      readxl_1.3.1     
## [21] lifecycle_1.0.0   munsell_0.5.0     gtable_0.3.0      cellranger_1.1.0 
## [25] rvest_1.0.1       evaluate_0.14     knitr_1.33        tzdb_0.1.2       
## [29] parallel_4.1.1    fansi_0.5.0       broom_0.7.9       Rcpp_1.0.7       
## [33] scales_1.1.1      backports_1.2.1   vroom_1.5.4       jsonlite_1.7.2   
## [37] bit_4.0.4         fs_1.5.0          hms_1.1.0         digest_0.6.27    
## [41] stringi_1.7.5     grid_4.1.1        cli_3.0.1         tools_4.1.1      
## [45] magrittr_2.0.1    sass_0.4.0        crayon_1.4.1      pkgconfig_2.0.3  
## [49] ellipsis_0.3.2    xml2_1.3.2        reprex_2.0.1      lubridate_1.7.10 
## [53] rstudioapi_0.13   assertthat_0.2.1  rmarkdown_2.10    httr_1.4.2       
## [57] R6_2.5.1          compiler_4.1.1</code></pre>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="unnumbered">References</h1>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-bipartite" class="csl-entry">
Dormann, C., B. Gruber, and J. Fründ. 2008. <span>“Introducing the Bipartite Package: Analysing Ecological Networks.”</span> <em>R News</em> 8: 8.
</div>
<div id="ref-mcfrederick2019wild" class="csl-entry">
McFrederick, Quinn S, and Sandra M Rehan. 2019. <span>“Wild Bee Pollen Usage and Microbial Communities Co-Vary Across Landscapes.”</span> <em>Microbial Ecology</em> 77 (2): 513–22.
</div>
<div id="ref-roughgarden1979theory" class="csl-entry">
Roughgarden, Joan. 1979. <em>Theory of Population Genetics and Evolutionary Ecology: An Introduction</em>.
</div>
<div id="ref-schoener1968anolis" class="csl-entry">
Schoener, Thomas W. 1968. <span>“The Anolis Lizards of Bimini: Resource Partitioning in a Complex Fauna.”</span> <em>Ecology</em> 49 (4): 704–26.
</div>
<div id="ref-zaccarelli2013ri" class="csl-entry">
Zaccarelli, Nicola, Daniel I Bolnick, and Giorgio Mancinelli. 2013. <span>“RI n s p: An r Package for the Analysis of Individual Specialization in Resource Use.”</span> <em>Methods in Ecology and Evolution</em> 4 (11): 1018–23.
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
